zscan_rr.go

```diff
@@ -2170,10 +2171,59 @@ func setIPSECKEY(h RR_Header, c chan lex, o, f string) (RR, *ParseError, string)
 	return rr, nil, c1
 }
 
+func setCAA(h RR_Header, c chan lex, o, f string) (RR, *ParseError, string) {
+	rr := new(CAA)
+	rr.Hdr = h
+	l := <-c
+	if l.token != "\\#" {
```

Comment:
you are parsing this is an unknown RR (see `setRFC3597`). This should be able to parse
`0 issue "ca.example.net; account=230123"` and `128 tbs "Unknown"`